signac/core/mongodbcollection.py

```diff
@@ -0,0 +1,147 @@
+# Copyright (c) 2020 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+"""Implements Mongo-backend.
+
+This implements the Mongo-backend for SyncedCollection API by
+implementing sync and load methods.
+"""
+from .synced_collection import SyncedCollection
+from .syncedattrdict import SyncedAttrDict
+from .synced_list import SyncedList
+
+
+class MongoCollection(SyncedCollection):
```

Comment:
```suggestion
class MongoDBCollection(SyncedCollection):
```
Please use the name "MongoDB" and its translation into code objects correctly and consistently.